internal/schemas.ts

```diff
@@ -843,6 +843,44 @@ Specifically, the requirements for different \`format\` values are:
   ],
 };
 
+const CompressedAudio: FoxgloveMessageSchema = {
+  type: "message",
+  name: "CompressedAudio",
+  description: "A single frame of a compressed audio bitstream",
+  fields: [
+    {
+      name: "timestamp",
+      type: { type: "primitive", name: "time" },
+      description: "Timestamp of audio frame",
+    },
+    {
+      name: "data",
+      type: { type: "primitive", name: "bytes" },
+      description: `Compressed audio frame data.`,
+    },
+    {
+      name: "format",
+      type: { type: "primitive", name: "string" },
+      description: "Audio format.\n\nSupported values: `opus`.",
+    },
+    {
+      name: "type",
+      type: { type: "primitive", name: "string" },
+      description: "Frame type.\n\n Supported values: `key`, `delta`.",
+    },
```

Comment:
We historically chose not to include this for CompressedVideo, although I don't think I know the whole reasoning behind the decision. It's certainly caused us a lot of pain having to detect key frames ourselves, but on the other hand it's nicer for users. cc @defunctzombie @snosenzo for any thoughts / additional historical context

Reply:
In practice with the MCAP recording demo as it is in https://github.com/foxglove/mcap/pull/1293, the frames are all keyframes, which would make this field somewhat unnecessary. That might not be the case for customers though.

It definitely pains me to include this extra field in every message (more data => more storage costs), but it does mean there is no logic of detecting keyframes like you said. A cursory search did not come up with how to detect audio keyframes, so I'd be worried about the complexity of the logic.